doc/man3/OSSL_HTTP_parse_url.pod

```diff
@@ -28,6 +32,19 @@ L<openssl_user_macros(7)>:
 
 =head1 DESCRIPTION
 
+OSSL_HTTP_adapt_proxy() takes an optional proxy hostname I<proxy>
+and returns it transformed according to the optional I<no_proxy> parameter,
+I<server>, I<use_ssl>, and the applicable environment variable, as follows.
+If I<proxy> is NULL, take any default value from the C<http_proxy>
+environment variable, or from C<https_proxy> if I<use_ssl> is nonzero.
+If this still does not yield a proxy hostname,
+take any further default value from the C<HTTP_PROXY>
+environment variable, or from C<HTTPS_PROXY> if I<use_ssl> is nonzero.
```

Comment:
Inconsistent spelling `C<http(s)_proxy>` vs. `C<HTTP(S)_PROXY>`

Reply:
No, this is on purpose: first try `http(s)_proxy`, then `HTTP(S)_PROXY`.
As discussed elsewhere, this is for compatibility with other HTTP clients such as `wget` and `cURL`.